examples/06-verif-manual/vm-009-large_lateral_deflection_of_unequal_stiffness_springs.py

```diff
@@ -382,20 +382,4 @@
 df2 = pd.DataFrame(main_columns, index=row_names)
 
 # Apply style settings for the dataframe.
-df2.style.set_table_styles([
-                            {
-                              "selector": "th",
-                              "props": [('font-size', '16px')]
-                            },
-                            {
-                              "selector": "td",
-                              "props": [('font-size', '16px')]
-                            },
-                            {
-                                "selector": "td:hover",
-                                "props": [("background-color", "#FFF8DC")]
-                            }]).set_properties(**
-                                               {
-                                                    "color": "black",
-                                                    "text-align": "center"
-                                               }).format("{:.2f}")
+df2.style.format("{:.2f}")
```

Comment:
Better use ``df.head()``, if you have many decimals in ``simulation_res`` you can do something like:

```python
simulation_res = np.asarray([strain_energy, disp_x, disp_y]).round(decimals=3)
```

Reply:
Hi @germa89 

I would like to offer use then 
```
# Create and fill the output dataframe with pandas.
df2 = pd.DataFrame(main_columns, index=row_names).round(2)
```
Because :
`simulation_res = np.asarray([strain_energy, disp_x, disp_y]).round(decimals=3)`
this line will control just a column of data table where we are applying the results from pymapdl, instead of control the decimal quantity of all data frame.

Reply:
Hi Roman

Since you are hard coding the target results:
```python
# Define the values of the target results.
target_res = np.asarray([24.01, 8.631, 4.533])
```
I though you were happy with their accuracy. 

But this is a very good suggestion. I'm happy with it.